tests/shipperCustomer.js

```diff
@@ -1,36 +1,34 @@
   await page.frameLocator('#iframe').getByRole('combobox', { name: 'Select a Shipper' }).click();
-  await page.frameLocator('#iframe').getByRole('option', { name: 'aci' }).click();
+  await page.frameLocator('#iframe').getByRole('option', { name: 'Alreach' }).click();
   await page.frameLocator('#iframe').getByPlaceholder('Reference Id').click();
-  await page.frameLocator('#iframe').getByPlaceholder('Reference Id').fill('123');
+  await page.frameLocator('#iframe').getByPlaceholder('Reference Id').press('CapsLock');
+  await page.frameLocator('#iframe').getByPlaceholder('Reference Id').fill('ka-tl');
   await page.frameLocator('#iframe').getByPlaceholder('Name', { exact: true }).click();
-  await page.frameLocator('#iframe').getByPlaceholder('Name', { exact: true }).fill('shipCust');
-  await page.frameLocator('#iframe').getByLabel('Customer Since').click();
-  await page.frameLocator('#iframe').getByText('NotesNotesCSR ReminderCustomer SinceLast Ordered').click();
-  await page.frameLocator('#iframe').locator('#pv_id_21 svg').click();
-  await page.frameLocator('#iframe').getByRole('option', { name: 'AK' }).click();
-  await page.frameLocator('#iframe').getByRole('button', { name: 'Save & Close', exact: true }).click();
-  await page.frameLocator('#iframe').getByRole('button', { name: 'SAVE & CLOSE', exact: true }).click();
+  await page.frameLocator('#iframe').getByPlaceholder('Name', { exact: true }).fill('Fodcrick');
   await page.frameLocator('#iframe').getByPlaceholder('1234 Street Name').click();
-  await page.frameLocator('#iframe').getByPlaceholder('1234 Street Name').fill('123 st mjahs');
+  await page.frameLocator('#iframe').getByPlaceholder('1234 Street Name').fill('Grand Canyon, 16 Wide weat st,.');
   await page.frameLocator('#iframe').getByPlaceholder('City').click();
-  await page.frameLocator('#iframe').getByPlaceholder('City').fill('newyork');
+  await page.frameLocator('#iframe').getByPlaceholder('City').fill('NEW YORK');
   await page.frameLocator('#iframe').getByPlaceholder('12345').click();
-  await page.frameLocator('#iframe').getByPlaceholder('12345').fill('12345');
-  await page.frameLocator('#iframe').getByRole('button', { name: 'SAVE & CLOSE', exact: true }).click();
-
-  
+  await page.frameLocator('#iframe').getByPlaceholder('12345').fill('187945');
+  await page.frameLocator('#iframe').getByRole('combobox', { name: 'AK' })
+  await page.frameLocator('#iframe').getByRole('option', { name: 'AL' }).click();
+  await page.frameLocator('#iframe').getByRole('button', { name: 'SUBMIT' }).click();
+  await page.frameLocator('#iframe').getByText('Shipper Customers GENERALAREASROUTING OVERRIDESADDRESSLABELSWAYPOINTSRATESNAVIGA').click();
+  //await page.frame({
+   // name: 'iframe'}).locator('div').filter({ hasText: 'Customer CreationCustomer Created With Success' }).nth(3).click();
+  await page.frameLocator('#iframe').getByRole('columnheader', { name: 'Customer Id' }).locator('svg').first().click();
```

Comment:
Assertions are needed to be expressed to check the outputs and expected results for the testcases; also having a screenshot for the actual result will be an added value.